src/framework/components/screen/component.js

```diff
@@ -231,13 +242,19 @@ class ScreenComponent extends Component {
         this._elements.forEach(element => element._onScreenSpaceChange());
     }
 
+    /**
+     * Gets whether the ScreenComponent will render its child {@link ElementComponent}s in screen
+     * space instead of world space.
```

Comment:
I saw you added dashes to "object space" everywhere, should screen space and world space have the same treatment?

Reply:
I've just done a Google search and it seems more common to not use a hyphen. So I've backed all that out.

Reply:
Thank you, there still is `object-space` now?

Reply:
Oh yeah! Forgot about that one. Thanks. Fixed.